ingestion_server/ingestion_server/indexer.py

```diff
@@ -346,6 +342,7 @@ def create_filtered_index(self, model_name: str, index_suffix: str, **_):
                 "dest": {"index": destination_index},
             }
         )
+        self.refresh(index_name=destination_index, change_settings=True)
```

Comment:
The only real change in this PR is this bit.

Reply:
The rest was needed for the green checkmark 😆.

Reply:
Ahhh, thank you, Dhruv!